samples/ControlCatalog/MainView.xaml.cs

```diff
@@ -13,11 +13,20 @@ public MainView()
         {
             this.InitializeComponent();
             if (AvaloniaLocator.Current.GetService<IRuntimePlatform>().GetRuntimeInfo().IsDesktop)
-                ((IList) this.FindControl<TabControl>("Sidebar").Items).Add(new TabItem()
-                {
-                    Header = "Dialogs",
-                    Content = new DialogsPage()
-                });
+            {
+                IList tabItems = ((IList)this.FindControl<TabControl>("Sidebar").Items);
+                tabItems.Add(new TabItem()
+                                                                           {
```

Comment:
Is this meant to be indented this far?

Reply:
This is how Rider does it, it indents to the same column. I prefer it this way but I don't mind changing it back.